languages/Dockerfile.builder.j2

```diff
@@ -3,25 +3,18 @@ FROM {{base_image}}
 # Set options that should be defined everywhere
 ENV JAVA_TOOL_OPTIONS=-Dfile.encoding=UTF8
 ENV LANG C.UTF-8
+ENV LANGPACKS_VERSION={{langpacks_version}}
```

Comment:
because i felt there was otherwise no way to tell what version of langpacks a given image was built from (something that can be annoying now as well).  if you exec into a built image - what version of the template files are in there?